src/nginxconfig/util/logging.js

```diff
@@ -24,10 +24,7 @@ OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
 THE SOFTWARE.
 */
 
-export const getAccessLogDomainPath = (domain, global) => {
-    return global.logging.accessLog.computed.replace(/([^/]+)\.log$/, `${domain.server.domain.computed}.$1.log`);
-};
+export const accessLogParamsDefault = 'buffer=512k flush=1m';
 
-export const getErrorLogDomainPath = (domain, global) => {
-    return global.logging.errorLog.computed.replace(/([^/]+)\.log (.+)$/, `${domain.server.domain.computed}.$1.log $2`);
-};
+export const errorLogLevelDefault = 'warn';
+export const errorLogLevelOptions = Object.freeze(['debug', 'info', 'notice', 'warn', 'error', 'crit', 'alert', 'emerg']);
```

Comment:
```suggestion
export const errorLogLevelOptions = Object.freeze(['debug', 'info', 'notice', 'warn', 'error', 'crit', 'alert', 'emerg']);

```
Newline eof, please

Reply:
also added an eslint rule to enforce it (c240490)